api/v1alpha2/cloudprovider_types.go

```diff
@@ -234,3 +242,34 @@ type CPILabelConfig struct {
 	// +optional
 	Region string `gcfg:"region,omitempty" json:"region,omitempty"`
 }
+
+// CPIRouteConfig is the vSphere cloud provider's route configuration.
+type CPIRouteConfig struct {
+	// T1RouterID is NSX-T T1 logical router ID used to create static route on Tier1 gateway.
+	// +optional
+	T1RouterID string `gcfg:"t1-router-id,omitempty" json:"t1RouterID,omitempty"`
```

Comment:
Later maybe we'll support Tier0 router, so do you think we could just use `RouterID`?
Another thing is right now I'm getting router `path` instead of `ID` from configuration, `path` could be used to identify it is Tier1 or Tier0 router. Do you think it's OK?

Reply:
Agreed, this should match the config from the cloud provider

Reply:
+1, yeah will update!